core/search_test.go

```diff
@@ -3,69 +3,109 @@ package core
 
-	mr, err := testrepo.NewTestRepo(nil)
+func TestSearch(t *testing.T) {
+	rc, _ := regmock.NewMockServer()
+	mr, err := testrepo.NewTestRepo(rc)
 	if err != nil {
 		t.Errorf("error allocating test repo: %s", err.Error())
 		return
 	}
 
+	cases := []struct {
```

Comment:
If there's only 1 search case we can drop the table test pattern & just run a raw test without the `cases` struct & loop.